Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -477,14 +477,9 @@ void WebAutomationSession::switchToBrowsingContext(const Inspector::Protocol::Au
     m_client->requestSwitchToPage(*this, *page, [frameID, page = Ref { *page }, callback = WTFMove(callback)]() mutable {
         page->setFocus(true);
 
-        if (!frameID) {
-            callback({ });
-            return;
-        }
-
-        ASYNC_FAIL_WITH_PREDEFINED_ERROR_IF(!WebFrameProxy::webFrame(frameID.value()), FrameNotFound);
+        ASYNC_FAIL_WITH_PREDEFINED_ERROR_IF(frameID && !WebFrameProxy::webFrame(frameID.value()), FrameNotFound);
```

Comment:
Hmmm.. how do we not fail this variant when `frameID == std::nullopt`?

Reply:
Oh, of course, `frameID && !WebFrameProxy::webFrame(frameID.value())` is the _error_ condition. Nvm